pull_request_template.md

```diff
@@ -1,12 +1,23 @@
-<!-- Description of the change, references like Slack or Trello links can provide more context -->
-
-### Friendly reminders
-- [ ] Describe the problem / feature (ideally with [meaningful commit messages](https://tekin.co.uk/2019/02/a-talk-about-revision-histories))
-- [ ] Lint rules pass
-- [ ] The code changed/added as part of this pull request has been covered with tests
-- [ ] The environment (`heroku config`) has been updated if needed (new `ENV` variables)
-
-### Type of change
-- [ ] Bug fix (non-breaking change that fixes an issue)
-- [ ] New feature (non-breaking change that adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+### WHY are these changes introduced?
+
+Please fill me in.
+
+### WHAT is this pull request doing?
+
+Please fill me in.
+
+### Changelog to support
+
+Please fill me in.
+
+### Reference to Trello card
+
+Please fill me in.
+
+---
```

Comment:
Just remembered something I like from template from previous workplace:

```
### How to test the PR

Please fill me in.
```

In general it would be "Specs cover", but in cases where it wasn't so simple we put in a small guide on how to use the feature etc so the reviewer could easily reproduce.

Reply:
I like this, I do, but the template is starting to get long now... hmm.

Reply:
To save some space, I wonder if we can "demote" `Reference to Trello card` from its own section to be part of the introduction instead, kind of similar to what we had before:

```markdown
<!-- Description of the change, references like Slack or Trello links can provide more context -->
[Trello](https://trello.com/c/...) or [Slack](https://84codes.slack.com/archives/...)
```

Reply:
I included it with https://github.com/84codes/.github/commit/2e0b7d3f46200cc5801c65fa81653983ae2be1dd, any feedback appreciated